.evergreen/config.yml

```diff
@@ -1320,14 +1349,47 @@ tasks:
         script: |-
           set -o errexit
           ${PREPARE_SHELL}
-          git add .
-          git commit -m "add files"
-          export AZUREOIDC_DRIVERS_TAR_FILE=/tmp/mongo-rust-driver.tgz
-          git archive -o $AZUREOIDC_DRIVERS_TAR_FILE HEAD
-          export AZUREOIDC_TEST_CMD="PROJECT_DIRECTORY='.' ./.evergreen/install-dependencies.sh rust\
-                                     && PROJECT_DIRECTORY='.' .evergreen/install-dependencies.sh junit-dependencies\
-                                     && PROJECT_DIRECTORY='.' OIDC_ENV=azure OIDC=oidc  ./.evergreen/run-mongodb-oidc-test.sh"
-          bash $DRIVERS_TOOLS/.evergreen/auth_oidc/azure/run-driver-test.sh
+           ./.evergreen/install-dependencies.sh rust
+          source .cargo/env
+          export AZUREOIDC_DRIVERS_TAR_FILE=/tmp/mongo-rust-driver.tar
+          rustup default stable
+          export RUSTFLAGS="-C target-feature=+crt-static"
+          cargo test test::spec::oidc::azure --features azure-oidc --no-run --target x86_64-unknown-linux-gnu
```

Comment:
suggest using the [`get_exe_name`](https://github.com/mongodb/mongo-rust-driver/blob/31a0750eecd61fc3f9251d378bc86f6dbf8d013f/src/test/atlas_planned_maintenance_testing.rs#L33) utility to retrieve the name of the test executable, example usage [here](https://github.com/mongodb/mongo-rust-driver/blob/31a0750eecd61fc3f9251d378bc86f6dbf8d013f/.evergreen/config.yml#L1725)

Reply:
I like this